src/app/screens/Home/index.tsx

```diff
@@ -286,45 +326,88 @@ function Home() {
         ) : (
           <div>
             <h2 className="mb-2 text-lg text-gray-900 font-bold dark:text-white">
-              Recent Transactions
+              {t("default_view.recent_transactions")}
             </h2>
-            {payments.length > 0 ? (
-              <TransactionsTable
-                transactions={payments.map((payment) => ({
-                  ...payment,
-                  type: "sent",
-                  date: dayjs(payment.createdAt).fromNow(),
-                  title: (
-                    <p className="truncate">
-                      <a
-                        target="_blank"
-                        title={payment.name}
-                        href={`options.html#/publishers`}
-                        rel="noreferrer"
-                      >
-                        {payment.name}
-                      </a>
+
+            <Tab.Group onChange={onTabChangeHandler}>
+              <Tab.List className="mb-2">
+                {[
+                  t("transaction_list.tabs.outgoing"),
+                  t("transaction_list.tabs.incoming"),
+                ].map((category) => (
+                  <Tab
+                    key={category}
+                    className={({ selected }) =>
+                      classNames(
+                        "w-1/2 rounded-lg py-2.5 font-bold transition duration-150",
+                        "focus:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 focus-visible:ring-orange-bitcoin",
+                        "hover:bg-gray-50 dark:hover:bg-surface-16dp",
+                        selected
+                          ? "text-orange-bitcoin"
+                          : "text-gray-700  dark:text-neutral-200"
+                      )
+                    }
+                  >
+                    {category}
```

Comment:
I have here two different styles:

default when I open the popup
<img width="441" alt="image" src="https://user-images.githubusercontent.com/318/180128342-29dc0347-0a6a-4410-92a8-8ca053e67b1b.png">
when I clock in incoming: 
<img width="399" alt="image" src="https://user-images.githubusercontent.com/318/180128436-e9ac682f-afaf-4aee-a98b-6ada510f426b.png">

Reply:
any objections to remove it?

Reply:
Done